var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -54,7 +55,11 @@ class Adios2(CMakePackage, CudaPackage):
     # change how we're supporting differnt library types in the package at anytime if
     # spack decides on a standardized way of doing it across packages
     variant("shared", default=True, when="+pic", description="Build shared libraries")
-    variant("pic", default=True, description="Build pic-enabled static libraries")
+    variant(
+        "pic",
+        default=not sys.platform == "win32",
```

Comment:
I wonder if a conflict() will be better than this. I am not 100% sure it is the case.

Reply:
I'm going to move this to the same stanza that loops over the platforms as the concept doesn't exist on Windows.